src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
OK. I'm going to update and move the Sprite anchorPoint positioning logic into the Sprite class itself.